example/form.yml

```diff
@@ -4,6 +4,10 @@
     description: What operating system are you using?
     placeholder: macOS Big Sur
     value: operating system
+  custom: 
```

Comment:
We should do it more like this:
```
customAttributes:
      "[SELECTOR]":
           - "[ATTR_NAME]": "[ATTR_VALUE]"
```

example:
```
customAttributes:
      "[div.form-group input]":
           - "name": "test"
           - "css": "text-white bg-red"
```

Then is completely dynamical.